libs/category/src/filters/behaviors/apply/apply-request-to-filters.spec.ts

```diff
@@ -0,0 +1,17 @@
+describe('daffApplyRequestToFilters', () => {
+  it('should apply a request', () => {
+
+  });
+
+  it('should apply a toggle request', () => {
```

Comment:
I don't understand what this function has to do with a toggle request. Should this be Equal vs Range instead of toggle vs idempotent?